admin/src/components/list/index.js

```diff
@@ -109,19 +109,26 @@ export const ResultTable = styled.div`
     }
   }
   select.sort-options {
-    background: none; // remove the arrows added by sortOptions
+    background: transparent;
+    background-image: url("data:image/svg+xml;utf8,<svg fill='black' height='24' viewBox='0 0 24 24' width='24' xmlns='http://www.w3.org/2000/svg'><path d='M7 10l5 5 5-5z'/><path d='M0 0h24v24H0z' fill='none'/></svg>");
+    background-repeat: no-repeat;
+    background-position-x: 100%;
+    background-position-y: 5px;
+    border: 1px solid #dfdfdf;
+    border-radius: 2px;
+    margin-right: 2rem;
     position: absolute;
     top: 0.5rem;
     left: 0.5rem;
     outline: 0;
-    background-color: #fff;
+    background-color: #f7fafc;
     box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
     border: 0;
     border-radius: 6px;
```

Comment:
You forgot to remove the existing `border` and `border-radius` properties (so it's erased).
I guess we could remove: 

```
border: 0;
border-radius: 6px;
```

Or maybe just keep the `border-radius: 6px;` and remove `border-radius: 2px;`